wiki/_Sidebar.md

```diff
@@ -0,0 +1,20 @@
+**Mimiker Wiki**
+- [Home](https://github.com/hadarai/mimiker-wiki/wiki)
+
+**Building toolchain**
+- [GNU Toolchain](https://github.com/hadarai/mimiker-wiki/wiki/Compiler-toolchain)
+- [OpenOCD](https://github.com/hadarai/mimiker-wiki/wiki/OpenOCD-for-kernel-development)
+- [QUEMU](https://github.com/hadarai/mimiker-wiki/wiki/QEMU-for-kernel-development)
+
+**Coworking information**
+- [Coding conventions](https://github.com/hadarai/mimiker-wiki/wiki/Coding-conventions)
+- [Github issue PR labels](https://github.com/hadarai/mimiker-wiki/wiki/Github-issue-PR-labels)
+
+**Developing documents**
+- [Test infrastructure](https://github.com/hadarai/mimiker-wiki/wiki/Test-infrastructure)
+- [Debugging](https://github.com/hadarai/mimiker-wiki/wiki/Debugging)
+- [Libkern](https://github.com/hadarai/mimiker-wiki/wiki/Libkern-information)
+
+**Miscellaneous**
+- [Related scripts](https://github.com/hadarai/mimiker-wiki/wiki/Related-scripts)
+- [Running Doom](https://github.com/hadarai/mimiker-wiki/wiki/Running-DOOM-on-mimiker)
```

Comment:
Why wiki is located in https://github.com/hadarai instaed of https://github.com/cahirwpz?

Reply:
That's a very good point.